deepinpy/recons/recon.py

```diff
@@ -18,10 +18,20 @@
 from torch.optim import lr_scheduler 
 
 @torch.jit.script
-def calc_nrmse(gt, pred):
+def calc_nrmse_real(gt, pred):
     resid = pred - gt
     return (torch.real(opt.zdot_single_batch(resid)) / torch.real(opt.zdot_single_batch(gt))).sqrt().mean()
 
+def calc_nrmse_abs(gt, pred):
+    temp = torch.sqrt(torch.mean(torch.square(torch.abs(gt) - torch.abs(pred))))
+    rr = torch.max(torch.abs(gt)) - torch.min(torch.abs(gt))  # range
+    return torch.div(temp, rr)
+
+def calc_nrmse(gt, pred, nrmse_only_real):
```

Comment:
can you add a default value, i.e. `nrmse_only_real=True` ?
Also, perhaps it's clearer to use the variable `absval=False` and switch the logic